py-polars/polars/lazyframe/frame.py

```diff
@@ -670,9 +670,9 @@ def __str__(self) -> str:
 {self.explain(optimized=False)}\
 """
 
-    def __repr__(self) -> str:
+    def __repr__(self) -> Mapping[str, Union[DataTypeClass, DataType]]:
```

Comment:
the return type needs to be a `str`, not `Mapping`, could you update that please? might need to check if there's a test which needs updating too (and if there isn't one, to write one)